students/17th/team1/markkim/westagram/settings.py

```diff
@@ -0,0 +1,120 @@
+"""
+Django settings for westagram project.
+
+Generated by 'django-admin startproject' using Django 3.1.5.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/3.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/3.1/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/3.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = '3-0fem@v84%55y3-*g5w!-0hazn+l-tr+j=&!22dd(+8h1l$9r'
```

Comment:
```suggestion
SECRET_KEY = '3-0fem@v84%55y3-*g5w!-0hazn+l-tr+j=&!22dd(+8h1l$9r'
```
씨크릿 키는  중요정보이므로 깃에 공유되어서는 안됩니다. 따라서  manage.py가 위치하는 경로에 my_settings.py같은 파일을 만들어서 거기에 저장한뒤에 임포트 후 적용해야 합니다. 변경 부탁드립니다.

Reply:
수정했습니다!